prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.ts

```diff
@@ -278,47 +292,73 @@ export class PharmanetEnrolmentSummaryComponent extends BaseEnrolmentPage implem
           };
         }
         case CareSettingEnum.COMMUNITY_PHARMACIST: {
           return {
             setting: 'Community Pharmacy',
             settingPlural: 'Community Pharmacies',
             settingCode: careSetting.careSettingCode,
-            formControl: this.pharmacistEmails,
-            subheaderContent: `Enter the email of the PharmaNet administrator in your workplace. If you work at more than one workplace, include the email addresses for each PharmaNet administrator, separated by a comma.`
+            formArray: this.pharmacistEmails,
+            formArrayName: 'pharmacistEmails',
+            subheaderContent: `Send your approval to your community pharmacy's PharmaNet administrator (e.g. office manager). If you work in more than one clinic make sure you include every PharmaNet administrator's email. Your PharmaNet administrator(s) will contact you once your PharmaNet access has been set up.`
           };
         }
         case CareSettingEnum.HEALTH_AUTHORITY: {
           return {
             setting: 'Health Authority',
             settingPlural: 'Health Authorities',
             settingCode: careSetting.careSettingCode,
-            formControl: this.healthAuthorityEmails,
-            subheaderContent: `Enter the email of the PharmaNet administrator in your workplace. If you work at more than one workplace, include the email addresses for each PharmaNet administrator, separated by a comma.`
+            formArray: this.healthAuthorityEmails,
+            formArrayName: 'healthAuthorityEmails',
+            subheaderContent: `Send your approval to your health authority's PharmaNet administrator (e.g. office manager). If you work in more than one clinic make sure you include every PharmaNet administrator's email. Your PharmaNet administrator(s) will contact you once your PharmaNet access has been set up.`
           };
         }
         case CareSettingEnum.DEVICE_PROVIDER: {
           return {
             setting: 'Device Provider',
             settingPlural: 'Device Providers',
             settingCode: careSetting.careSettingCode,
-            formControl: this.deviceProviderEmails,
-            subheaderContent: `Enter the email of the PharmaNet administrator in your workplace. If you work at more than one workplace, include the email addresses for each PharmaNet administrator, separated by a comma.`
+            formArray: this.deviceProviderEmails,
+            formArrayName: 'deviceProviderEmails',
+            subheaderContent: `Send your approval to your device provider's PharmaNet administrator (e.g. office manager). If you work in more than one clinic make sure you include every PharmaNet administrator's email. Your PharmaNet administrator(s) will contact you once your PharmaNet access has been set up.`
           };
         }
       }
     });
   }
 
-  private buildVendorEmailGroup(): FormGroup {
+  protected addEmail(emailsGroup: FormArray, email?: string): void {
```

Comment:
Rather than `emailsGroup`, is `emailsArray` better?

Reply:
hm, kindda picky. I guess why not.